mesh/nurbs.hpp

```diff
@@ -59,9 +59,12 @@ class KnotVector
        integers are read, for order and number of control points. */
    KnotVector(std::istream &input);
 
-   /** @brief Create a KnotVector with undefined knots (initialized to -1) of
-       order @a order and number of control points @a NCP. */
-   KnotVector(int order, int NCP);
+   /** @brief Create a KnotVector with order @a order.
+       When @a NCP is not provided  the number of control points is set to
```

Comment:
Extra space, typo in `respectively`. Actually, the word `respectively` is unnecessary here.